website/views/shared/modals/quest-rewards.jade

```diff
@@ -12,3 +12,8 @@ script(id='partials/options.social.party.quest-rewards.html', type='text/ng-temp
         =env.t('gold')
     tr(ng-if='::quest.drop.unlock()')
       td {{::quest.drop.unlock()}}
+  div(ng-if='_.filter(quest.drop.items, \'onlyOwner\').length > 0')
```

Comment:
One last comment for this PR. Can you move `_.filter(quest.drop.items, \'onlyOwner\')` to a variable on the controller. OR, since this is a model, move it to the Modal scope?